### ISSUE TITLE
Chapter 5 and the connection to generative models

### ISSUE THREAD

**Issue by chrarnold** [udlbook/udlbook#163]
The participants, including me, from our Book Reading Club were a bit confused about Chapter 5 and the way it was structured. We got the impression that the neural networks that were discussed and how they work (i.e., one output value per distribution parameter) can somewhat be seen as generative models because of the explicit distributions that are decided in the beginning how to model y. To what extend can that be seen as such?

Figure 5.4, with optimizing either the horizontal or vertical distances, also reminded us of Problem 2.3.

Thus in summary, maybe an explicit mentioning of this apparent connection may be useful. And maybe here for all who wonder ;)

**Comment by udlbook** [comment]
Hi Christian,

Apologies for the long delay.  I've just returned from vacation.  You are
correct, they are similar to generative models.  However, the loss function
is different.  Here, the network predicts one probability distribution for
each input x and tries to make the matching data y as probable as possible
under that distribution.   Generative models are essentially a black box
that describe the probability of all of the data at once (i.e. we want to
match the probability distribution).   However, this is somewhat muddied
when we use generative models with latent variables.

I don't really want to get into this at this point in the book because at
this point, the reader doesn't necessarily know what generative models
are.  However, I can see that there is a good case for having a problem in
the generative models chapter that investigates this issue.  I have flagged
this for a change in the next edition.

The horizontal distance in figure 5.4 is a bit of a red herring;  it's just
the only way that I can draw it.  It might be better to think of the
Gaussians in that figure as sticking out of the page and the associated
dashed lines as pointing towawrds you.

Simon
